protos/feast/serving/ServingService.proto

```diff
@@ -81,6 +84,23 @@ message GetOnlineFeaturesRequestV2 {
     }
 }
 
+message FeatureListConfig {
```

Comment:
Config is a bit confusing to me. To me it speaks more to load time configuration than request time options. I also really like the idea of less nested structures in the Json.

What if we used `repeated FeatureReferenceV2 features` and pulled out `full_feature_names` to the top level? Would that be more in line with our `get_online_features()` API?

Reply:
I'm changing this, so I'll resolve this comment. Please take a look at the new API and let me know if you have any comments on that.